test/extensions/access_loggers/grpc/http_grpc_access_log_impl_test.cc

```diff
@@ -127,7 +141,11 @@ TEST_F(HttpGrpcAccessLogTest, Marshalling) {
     stream_info.last_downstream_tx_byte_sent_ = 2ms;
     stream_info.setDownstreamLocalAddress(std::make_shared<Network::Address::PipeInstance>("/foo"));
     (*stream_info.metadata_.mutable_filter_metadata())["foo"] = ProtobufWkt::Struct();
-
+    stream_info.filter_state_.setData("string_accessor",
```

Comment:
Do you have full test coverage of all the new code and cases? E.g. covering the Any swap vs. pack behavior.

Reply:
Yes: https://295805-65214191-gh.circle-artifacts.com/0/coverage/coverage/proc/self/cwd/source/extensions/access_loggers/grpc/grpc_access_log_utils.cc.html#L248